src/DIRAC/Core/scripts/dirac_configure.py

```diff
@@ -593,6 +593,8 @@ def runDiracConfigure(params):
         mkDir(configDir)
         params.update = True
         DIRAC.gConfig.dumpLocalCFGToFile(params.outputFile)
+    else:
+        DIRAC.gLogger.info(f"{params.outputFile} exists, not overwriting it.")
```

Comment:
```suggestion
        DIRAC.gLogger.notice(f"{params.outputFile} exists, not overwriting it. Or use the '--ForceUpdate' Flag")
```
* Very good addition!
* Info still doesn't print in normal script use, does it? (neither does warn??) So have to use `notice`...
* ForceUpdate still works, right?

Reply:
indeed ! Thanks